node/resolvers/session/sessionResolver.ts

```diff
@@ -44,6 +45,12 @@ interface UtmiParams {
 
 const convertToBool = (str: any) => !!str && toLower(str) === 'true'
 
+const priceTables = (profile: SessionProfile): string[] => {
+  const priceTables =
+    profile && profile.priceTables && profile.priceTables.value
```

Comment:
The value comes as a string with commas.

Example:
```
"priceTables": {
  "value": "vip,1"
}
```
This function change it to an array of strings